core/iwasm/aot/aot_loader.c

```diff
@@ -1684,6 +1684,11 @@ load_types(const uint8 **p_buf, const uint8 *buf_end, AOTModule *module,
 
             func_type->ref_type_map_count = ref_type_map_count;
 
+#if WASM_ENABLE_FUZZ_TEST != 0
```

Comment:
NOT sure `#if ... #endif` is necessary. But it might be helpful for potential performance concern.